### ISSUE TITLE
按照RGBT-GroundBench-main/script_train/RGBT_VGNet配置训练出现训练坍塌

### ISSUE THREAD

**Issue by leileilei2000** [crazyxiaoxi/RGBT-GroundBench#5]
Namespace(modality='rgbt', open_lora=True, open_text_guided_fusion=True, lavs_mode='lavs', lora_r_rgb=16, lora_r_ir=48, sup_type='full', lr=0.0005, batch_size=16, weight_decay=0.0001, epochs=60, lr_power=0.9, lr_exponential=0.9, clip_max_norm=0.0, eval=False, optimizer='adamw', lr_scheduler='cosine', lr_drop=60, aug_blur=False, aug_crop=True, aug_scale=True, aug_translate=True, model='ViT-B/16', model_name='MMVGFusion', extract_layer=0, warmup=False, dilation=False, position_embedding='sine', dim_feedforward=2048, dropout=0.1, nheads=8, num_queries=100, pre_norm=False, imsize=224, emb_size=512, vl_dropout=0.1, vl_nheads=8, vl_hidden_dim=512, vl_dim_feedforward=2048, vl_enc_layers=6, vl_dec_layers=6, data_root='../dataset_and_pretrain_model/datasets/VG/image_data', split_root='../dataset_and_pretrain_model/datasets/VG/ref_data_shuffled', dataset='rgbtvg_m3fd', max_query_len=77, prompt='', use_cot_prompt=False, cot_length=0, use_contrastive_loss=True, use_rtcc_constrain_loss=True, use_mask_loss=True, use_seg_mask=False, retrain='../dataset_and_pretrain_model/pretrain_model/pretrained_weights/MMVG/mixup_pretraining_base/mixup/merged_fixed_best_checkpoint_peft0111.pth', adapt_mlp=False, normalize_before=True, save_hilora_clip=False, hi_lora_stage=0, hi_lora_retrain='', hi_lora_clip='', mixup_pretrain=True, enable_adaptive_weights=False, FusionMethod='IAFv3', cross_num_attention_heads=1, cross_vis_hidden_size=512, cross_text_hidden_size=512, cross_hidden_dropout_prob=0.1, cross_attention_probs_dropout_prob=0.1, output_dir='./output_retraining/MMVGFusion_224_rgbt/rgbtvg_m3fd/rgbt_finetuning_base_clip_weight/lavs/lora_r16_48/output_v100', device='cuda', seed=13, resume='', clip_model='', bert_model='bert-base-uncased', light=False, start_epoch=0, num_workers=4, world_size=2, dist_url='env://', rank=0, gpu=0, distributed=True, dist_backend='nccl')
{"epoch": 0, "train_lr": 0.0005000000000000003, "train_loss": 5.293351377759661, "train_loss_algin_focal": 0.21191390705228383, "train_loss_align_dice": 0.40943168236740995, "train_loss_bbox": 0.4718744906463793, "train_loss_contrastive": 1.0242908533130373, "train_loss_giou": 1.782746210694313, "train_loss_seg_dice": 1.1957165428570338, "train_loss_seg_focal": 0.19737772743350693, "validation_mask seg miou": 0.44461861110868905, "validation_miou": 0.34796819232759024, "validation_accu": 0.39880952380952384, "n_parameters": 266462996}
{"epoch": 1, "train_lr": 0.0004996573836886438, "train_loss": 4.015580204980714, "train_loss_algin_focal": 0.08195674584047603, "train_loss_align_dice": 0.3360693325687732, "train_loss_bbox": 0.3936322243618114, "train_loss_contrastive": 0.3212713698324348, "train_loss_giou": 1.7126448250242643, "train_loss_seg_dice": 1.0822175507034575, "train_loss_seg_focal": 0.0877881487166243, "validation_mask seg miou": 0.401771795182001, "validation_miou": 0.3368628138587588, "validation_accu": 0.3869047619047619, "n_parameters": 266462996}
{"epoch": 2, "train_lr": 0.0004986304738420691, "train_loss": 3.7794820921761647, "train_loss_algin_focal": 0.06999660848772951, "train_loss_align_dice": 0.327152215476547, "train_loss_bbox": 0.37584367554102627, "train_loss_contrastive": 0.22722518876461045, "train_loss_giou": 1.6751113459467888, "train_loss_seg_dice": 1.023895195552281, "train_loss_seg_focal": 0.0802578410865473, "validation_mask seg miou": 0.4547622317359561, "validation_miou": 0.3642653397151402, "validation_accu": 0.375, "n_parameters": 266462996}
{"epoch": 3, "train_lr": 0.0004969220851487857, "train_loss": 4.830703856689589, "train_loss_algin_focal": 0.15944102148724987, "train_loss_align_dice": 0.46657584687428816, "train_loss_bbox": 0.5666215032605189, "train_loss_contrastive": 0.27705906464585234, "train_loss_giou": 1.9342454937951905, "train_loss_seg_dice": 1.287077504077128, "train_loss_seg_focal": 0.139683425991929, "validation_mask seg miou": 0.02752435916946048, "validation_miou": 0.03347215624082656, "validation_accu": 0.0, "n_parameters": 266462996}
{"epoch": 4, "train_lr": 0.000494536900183452, "train_loss": 6.8575060069561005, "train_loss_algin_focal": 0.27429221597101006, "train_loss_align_dice": 0.7189496577318225, "train_loss_bbox": 0.9442163546170507, "train_loss_contrastive": 0.5938108846811312, "train_loss_giou": 2.3706316628626416, "train_loss_seg_dice": 1.8205808166946684, "train_loss_seg_focal": 0.13502440634848817, "validation_mask seg miou": 0.0, "validation_miou": 0.04420725646473113, "validation_accu": 0.011904761904761904, "n_parameters": 266462996}
{"epoch": 5, "train_lr": 0.0004914814565722677, "train_loss": 6.376502403191158, "train_loss_algin_focal": 0.2533812318662448, "train_loss_align_dice": 0.7208512893744877, "train_loss_bbox": 0.9421056503696101, "train_loss_contrastive": 0.18382917527508522, "train_loss_giou": 2.321948985968317, "train_loss_seg_dice": 1.8229100917066847, "train_loss_seg_focal": 0.1314759913366288, "validation_mask seg miou": 0.0, "validation_miou": 0.05282998085021973, "validation_accu": 0.005952380952380952, "n_parameters": 266462996}
{"epoch": 6, "train_lr": 0.0004877641290737881, "train_loss": 6.383743452174323, "train_loss_algin_focal": 0.23298232430326088, "train_loss_align_dice": 0.7156720472765821, "train_loss_bbox": 0.9047534428536892, "train_loss_contrastive": 0.283086854598618, "train_loss_giou": 2.296089979154723, "train_loss_seg_dice": 1.830403025661196, "train_loss_seg_focal": 0.12075583450496197, "validation_mask seg miou": 0.0, "validation_miou": 0.05571392320451282, "validation_accu": 0.005952380952380952, "n_parameters": 266462996}
{"epoch": 7, "train_lr": 0.00048339510662429925, "train_loss": 6.204600193670818, "train_loss_algin_focal": 0.21878468311790908, "train_loss_align_dice": 0.701252673885652, "train_loss_bbox": 0.8647009139614446, "train_loss_contrastive": 0.1452495940861159, "train_loss_giou": 2.313861372215407, "train_loss_seg_dice": 1.8332780950835772, "train_loss_seg_focal": 0.12747286725789309, "validation_mask seg miou": 0.0002242595711279483, "validation_miou": 0.07749332132793608, "validation_accu": 0.02976190476190476, "n_parameters": 266462996}
{"epoch": 8, "train_lr": 0.00047838636441065126, "train_loss": 6.091018778937204, "train_loss_algin_focal": 0.21874447047178233, "train_loss_align_dice": 0.7273084718201842, "train_loss_bbox": 0.8383958557886737, "train_loss_contrastive": 0.09359156523299005, "train_loss_giou": 2.2687008657625745, "train_loss_seg_dice": 1.8223739841154643, "train_loss_seg_focal": 0.12190357891709677, "validation_mask seg miou": 0.0, "validation_miou": 0.08908697820845105, "validation_accu": 0.047619047619047616, "n_parameters": 266462996}

**Comment by crazyxiaoxi** [comment]
The code has been updated. Please pull the latest version and try again.
